src/iexamine.cpp

```diff
@@ -2780,6 +2783,19 @@ void iexamine::water_source(player &p, const tripoint &examp)
     g->handle_liquid( water, nullptr, 0, &examp );
 }
 
+void iexamine::milk_source( monster *source_mon )
```

Comment:
>Use references instead of pointers where possible, unless you expect the monster to be null at times.

I'm very noob with C++ so I have absolutely not idea of what's the difference between using pointer and reference in this situation, so I just copied how the other situations with vehicles, item source and map worked. Would you mind explaining it please? @Coolthulhu

>It's not a huge error (just a matter of convention)

I don't understand what you mean by "convention", is there any info regarding conventions? as to not commit this mistake again.

> I'd allow it, but from what I know we're in feature freeze right now so you have time to fix it.

Then should I make a new pair of files or leave it as it is?

>The proper place for it would be either a new pair of files

I feel like you wanted to say something more(by that either) and got cut short.